spec.yaml

```diff
@@ -355,6 +355,47 @@ paths:
               type: string
           schema:
             $ref: "#/definitions/Error"
+  /invites/forward:
```

Comment:
Who forwards the invitation when this end-point is called, and from where to where?

Reply:
IIUC, the flow would be as follows, if Alice at server A invites Bob at server B:
* Alice instructs A to send an invite to "Bob@B"
  * A changes state, "invite sent from Alice to Bob@B"
  * A calls B/invites/forward
  * B changes state, "invite received from Alice@A to Bob"
* B notifies Bob (this is the "forwarding", right?)
* Bob instructs B to accept the invitation
  * B changes state, "invite from Alice@A to Bob accepted"
  * B calls A/invites/accept
  * A changes state, "invite from Alice to Bob@B accepted"

If this is correct, then I think the summary and description should reflect that it is B who forwards the invite from A (or effectively from Alice@A) to Bob?

Reply:
The summary "Forward an invitation to start sharing" suggests that the agent doing the forwarding (B) is also the agent who starts sharing, but I think it's more accurate to say Alice@A starts sharing.

Also, since B forwards from the inner network to the last mile end-point, "deliver" might have been a better verb than "forward".

The delivery of the notification is not the main functionality exposed by this API call; rather, if and how B notifies Bob is not even the concern of A or of Alice@A. So given that there is also a state change in B, maybe "create" (as in "create an entry in your list of received invites") would have been even better than "forward" or "deliver".

The description "Forwards an invitation received through an out-of-band channel, email for example." describes  what how the invitation was received, but I think the intention was to describe how the invitation gets forwarded.
So maybe "Forwards a received invitation through an out-of-band channel, email for example." is clearer?

Reply:
And for the summary I would suggest "Notify the recipient of a new invite"